swagger/public_v1/api_spec.json

```diff
@@ -529,6 +529,87 @@
           }
         }
       },
+      "LocationStatus": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string",
+            "example": "22329867"
+          },
+          "type": {
+            "type": "string",
+            "example": "site_statuses"
+          },
+          "attributes": {
+            "$ref": "#/components/schemas/LocationStatusAttributes"
+          }
+        }
+      },
+      "LocationStatusAttributes": {
+        "type": "object",
+        "properties": {
+          "status": {
+            "type": "string",
+            "enum": [
+              "discontinued",
+              "running",
+              "new_status",
+              "suspended"
+            ],
+            "example": "running"
+          },
+          "publish": {
+            "type": "string",
+            "enum": [
+              "published",
+              "unpublished"
+            ],
+            "example": "published"
+          },
+          "has_vacancies?": {
+            "type": "boolean",
+            "description": "Are there still slots available for this course?",
```

Comment:
```suggestion
            "description": "Are there still slots available at this location for this course?",
```

Reply:
not sure how slots came about

Reply:
sorry, this is me. feel free to amend. i still don't know what the domain terminology should be

Reply:
i've now removed slots